pkg/cloudprivateipconfig/cloudprivateipconfig.go

```diff
@@ -0,0 +1,40 @@
+package cloudprivateipconfig
+
+import (
+	"errors"
+	"net"
+	"strings"
+)
+
```

Comment:
nit: cloudprivateipconfig.go:9:6: exported type IPFamily should have comment or be unexported